tests/spreadsheet_test.py

```diff
@@ -36,10 +36,12 @@ def test_get_worksheet(self):
         sheet1 = self.spreadsheet.get_worksheet(0)
         self.assertTrue(isinstance(sheet1, gspread.Worksheet))
 
-    @pytest.mark.vcr()
+    @pytest.mark.vcr(allow_playback_repeats=True)
```

Comment:
with my advice (see [comment](https://github.com/burnash/gspread/pull/1269#issuecomment-1671494535)) you should not need this